src/vtkF3DRenderPass.cxx

```diff
@@ -254,8 +254,16 @@ void vtkF3DRenderPass::Blend(const vtkRenderState* s)
     {
       ssImpl << "  vec3 bgCol = texture(texBackground, texCoord).rgb;\n";
     }
-    ssImpl << "  vec3 result = bgCol * (1.0 - mainSample.a) + mainSample.rgb * mainSample.a;\n"
-              "  gl_FragData[0] = vec4(result.rgb, 1.0);\n";
+    ssImpl << "  vec3 result = bgCol * (1.0 - mainSample.a) + mainSample.rgb * mainSample.a;\n";
```

Comment:
using `mix` to interpolate would be cleaner imo: https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/mix.xhtml